src/index.html

```diff
@@ -17,6 +29,94 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          class="moyo_logo"
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="link_box">
+            <a
+              class="nav_link is-active"
+              href="#"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="link_box">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="link_box">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="link_box">
+            <a
+              class="nav_link"
+              data-qa="hover"
+              href="#"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="link_box">
+            <a
+              class="nav_link"
+              href="#"
```

Comment:
The `href` attribute for the navigation links is set to `#`, which is a placeholder. Consider replacing it with valid URLs or JavaScript functions to handle the click events if these are intended for navigation.